tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "ES2019",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
This will likely need to be adjusted. Is Node 10+ the requirement for this?

https://github.com/microsoft/TypeScript-wiki/blob/master/Node-Target-Mapping.md

Reply:
There's no exact Node.js version requirement or target version, I'm just trying to use newer features while not trying to require people to upgrade to Node.js 14.